baseband/vlbi_base/base.py

```diff
@@ -342,7 +342,7 @@ def _get_frame_rate(fh, header_template):
             fh.seek(header.payloadsize, 1)
             header = header_template.fromfile(fh)
 
-        if header.seconds != sec0 + 1:  # pragma: no cover
+        if header['seconds'] != sec0 + 1:  # pragma: no cover
```

Comment:
Aside: I think this part was the original reason I wrote the attribute->header item conversion: I had the idea that not every header might contain 'seconds', and that one could then work around it by defining an attribute. But I think it is clear now that this is not needed in practice.